Automattic/jetpack
# Conversation block: rename to Transcription block

#### Changes proposed in this Pull Request:

* Renames the "Conversation" block to "Transcription", including the folder name and any references to "conversation" in code
* Renames "Participant" to "Speaker" in the block UI, to be consistent (but all code reference continue to reference "participant")

The PR looks like a lot of changes, but everything is simply renaming, along with Prettier formatting updates.

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Insert a _transcription_ block, and verify adding speakers and dialogue works as before
* Search the code and make sure no references to "conversation" remain (other than keywords)

#### Proposed changelog entry for your changes:

* Conversation block: rename to "Transcription"
